mycore-solr/src/main/java/org/mycore/solr/commands/MCRSolrCoreAdminCommands.java

```diff
@@ -188,4 +194,33 @@ public static void reloadSolrConfiguration(String configType, String coreID) {
         MCRSolrSchemaReloader.processSchemaFiles(configType, coreID);
         MCRSolrConfigReloader.processConfigFiles(configType, coreID);
     }
+
+    @MCRCommand(
+            syntax = "export solr configset {0} to folder {1}",
+            help = "exports a solr configset to a folder",
+            order = 80)
+    public static void exportSolrConfigSet(String configSet, String folder) {
+        Path exportFolder = Paths.get(folder);
+        if(Files.notExists(exportFolder)) {
```

Comment:
Could also check if `exportFolder` is indeed a folder and actually writable

Reply:
isFolder() seems a good idea,
on the other side this command is for admins that usually understand the exception that is thrown if the folder is not writabe.

Reply:
Added isDirectory check. Checking if a file is writable is only possible if the file already exists.